Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs

```diff
@@ -23,6 +23,14 @@ static Task PlatformSetAsync(string key, string data)
         {
             try
             {
+                try
+                {
+                    // Swallow the exception since we are just removing it if it exists
+                    DataManager.RemoveAlias(key);
```

Comment:
Is it good to check if it exists first? I would also log Debug message on the catch.

Reply:
Re checking first, I wan't sure if we wanted to triple the cost of this call, instead of doubling as it stands - getting it throws an exception if it doesn't exist so you wouild be pretty much back to the current situation.

My thinking is that logging a failed delete will polute the log because it will be happening  all the time as values are overwritten. If I were to call PlatformRemove instead then that would do the log.  This is what I did in the earlier version, but decided to make it silent since it is not actually an error.  If you'd prefer I can log a message though - I'm not sure it is actionable.

Reply:
I'm open to implementing either of your suggestions after you've considered my reasoning.  Let me know.

Reply:
Exceptions aren't necessarily cheap either, so my gut says we should probably check before removing.

Reply:
> Exceptions aren't necessarily cheap either, so my gut says we should probably check before removing.

I know - that is why I'm not sure that checking makes sense because checking throws an exception if the key is not there.  I'll update the code as you suggest though :-)

Reply:
Let me know if you'd prefer that I use the native calls rather than the platform wrappers.

Reply:
This won't work, since the Tizen `PlatformGetAsync` looks to throw an Exception for non-existent values (a separate bug).  I'll update the code again.